### PR TITLE
Update system_requirements.adoc

### PR DESCRIPTION
**Pull request by MorShabiPAN**
updated the ARM support paragraph

@PaloAltoNetworks/announcements, @Pubs-MV,  @rcaballeromx  can you please review the following suggestions?

### CODE REVIEW

**Comment by MorShabiPAN** [comment]
Hi @Pubs-MV, @rcaballeromx can one of you please. review this suggestion?

Best,
Mor

**Comment by rcaballeromx** [comment]
Thank you @MorShabiPAN. I've improved upon your suggestion and added the needed markup to make the cross-references functional. PTAL
@harisrini: I've also clarified the Openshift 3.11 support issue you raised, PTAL
@MayaShani I included your change from [PR 377](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/377) PTAL

**Comment by MorShabiPAN** [comment]
Thanks @rcaballeromx

File: compute/admin_guide/install/system_requirements.adoc [hunk 1]
```diff
@@ -189,13 +187,14 @@ The following use cases are currently unsupported:
 ====
 
 |RHCOS
-|Openshift v4 versions
+|Openshift v4 versions, Openshift 3.11 is supported, but will be deprecated in the next release.
```
Comment (Pubs-MV) [inline compute/admin_guide/install/system_requirements.adoc]: @MorShabiPAN @rcaballeromx can we just list the versions in the system reqs without announcing deprecation in the admin guide. 
This is deprecation notice is included in the RN for 22.06.
The other reason for the omission is consistency, we also have other deprecation notices called out in the RN and we have not annotated it in the admin guide. If we decide to follow this approach, we need to make it uniform for all such changes.

Comment (rcaballeromx) [reply compute/admin_guide/install/system_requirements.adoc]: I've changed the text as you suggested @Pubs-MV

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**